src/System.Runtime.Extensions/tests/System.Runtime.Extensions.Tests.csproj

```diff
@@ -100,6 +101,10 @@
       <Project>{1e689c1b-690c-4799-bde9-6e7990585894}</Project>
       <Name>System.Runtime.Extensions.CoreCLR</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Runtime\pkg\System.Runtime.pkgproj">
+    </ProjectReference>
+    <ProjectReference Include="..\..\System.AppContext\pkg\System.AppContext.pkgproj">
```

Comment:
I get this error otherwise because now AppContext exists in both assemblies.
System\AppDomainTests.cs(29,65): error CS0433: The type 'AppContext' exists in both 'System.AppContext, Version=4.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' and 'System.Runtime, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' [F:\corefxgit\corefx\src\System.Runtime.Extensions\tests\System.Runtime.Extensions.Tests.csproj]